src/components/CsvUpload/index.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input } from '@edx/paragon';
+
+class CsvUpload extends React.Component {
```

Comment:
You may already be doing this, I know you said you were gonna push some tests, but can you add some tests for this component? Basically I want to test that it properly encodes / decodes the value.